chunker/pom.xml

```diff
@@ -19,7 +19,7 @@
         <dependency>
             <groupId>edu.illinois.cs.cogcomp</groupId>
             <artifactId>LBJava</artifactId>
-            <version>1.2.24</version>
+            <version>1.3.0</version>
```

Comment:
if we're upgrading lbjava version with pruning, ideally we should update all the model files with feature pruning. @mssammon what do you think? 
Btw did e update the parameters after we changed to `double`s to `float`s?

Reply:
There were some efforts on testing single-precision models but don't remember how things went.